EU4toV2/Source/V2World/Diplomacy/Diplomacy.cpp

```diff
@@ -1,5 +1,6 @@
 #include "Diplomacy.h"
 #include "../../Configuration.h"
+#include "../../EU4World/World.h"
```

Comment:
I believe @Zemurin has updated the paths so you can just do 
```
#include "EU4World/World.h"
```

Reply:
#include "World.h"
should be enough